modules/deploy/pages/deployment-option/cloud/vpc-byo-gcp.adoc

```diff
@@ -0,0 +1,488 @@
+= Configure a Customer-Managed VPC on GCP
+:description: Connect Redpanda Cloud to your existing VPC resources for additional security.
+:page-cloud: true
+
+You can connect Redpanda Cloud to your customer-managed virtual private cloud (VPC), and you can then create clusters with the most security. With standard BYOC clusters using a Redpanda-managed VPC, Redpanda creates a VPC in your Google Cloud account and manages its resources. With a customer-managed VPC:
+
+* You provide your own VPC in your Google Cloud account.
+* You maintain more control of your Google Cloud account, because Redpanda requires less permissions than standard BYOC clusters.
```

Comment:
```suggestion
* You maintain more control of your Google Cloud account, because Redpanda requires fewer permissions than standard BYOC clusters.
```

Reply:
Learned about this distinction in a college grammar class in college. For whatever reason, found it interesting and have always adhered to it:
"fewer" is used to refer to number among things that are counted, as in "fewer choices" and "fewer problems"; "less" is used to refer to quantity or amount among things that are measured, as in "less time" and "less effort."